website/content/docs/auth/gcp.mdx

```diff
@@ -292,7 +292,9 @@ For the API method, expiration is optional and will default to an hour.
 If specified, expiration must be a
 [NumericDate](https://tools.ietf.org/html/rfc7519#section-2) value (seconds from
 Epoch). This value must be before the max JWT expiration allowed for a role.
-This defaults to 15 minutes and cannot be more than 1 hour.
+This defaults to 15 minutes and cannot be more than 1 hour. 
+
+If a user generates a token that expires after 15 minutes and gcp role has `max_jwt_exp` set to default 15 minutes, Vault will return an error like `Expiration date must be set to no more that 15 mins in JWT_CLAIM, otherwise the login request returns error "role requires that service account JWTs expire within 900 seconds"`. In this case user must create a new signed JWT with a shorter expiration or set `max_jwt_exp` to higher value in gcp role.
```

Comment:
```suggestion
If a user generates a token that expires after 15 minutes, and gcp role has `max_jwt_exp` set to the default, Vault will return the error: `Expiration date must be set to no more that 15 mins in JWT_CLAIM, otherwise the login request returns error "role requires that service account JWTs expire within 900 seconds`. In this case, the user must create a new signed JWT with a shorter expiration, or set `max_jwt_exp` to a higher value in the gcp role.
```

Reply:
I modified this slightly but unsure of the error mentioned is the actual output of the error. The sentence "Vault will return an error like..." is indicating to me that it's not a definitive error message. However, if it is, then we'd want a definitive statement, "Vault will return the following error..."